pkg/graph/resolvers/system_intake_actions.go

```diff
@@ -951,6 +953,7 @@ func ConfirmLCID(ctx context.Context,
 	// update LCID-related fields
 	intake.TRBFollowUpRecommendation = &input.TrbFollowUp
 	intake.LifecycleExpiresAt = &input.ExpiresAt
+	intake.LifecycleExpirationAlertTS = nil // whenever we update intake.LifecycleExpiresAt (above), we should clear this field so alerts fire properly
```

Comment:
do we need to check that `intake.LifecycleExpiresAt != input.ExpiresAt`? Is it possible to do an LCID action that does not result in an updated expires at?

```
intake.LifecycleExpiresAt = &input.ExpiresAt
if !intake.LifecycleExpiresAt.Equal(input.ExpiresAt) {
    intake.LifecycleExpirationAlertTS = nil
}
```

Reply:
We could add that, but the SQL call that gets made is always going to set that column anyways, so we'd just be preventing setting the struct field to the same thing

Reply:
And since `input.ExpiresAt` is not nullable and is required, it's probably just easier/cleaner to just set it every time

Reply:
https://github.com/CMSgov/easi-app/pull/2429/commits/46acb722c8d4613cb438e73ebea71c598ef8b9be